tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/fallback/ConfigPropertyGlobalVsMethodTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.fault.tolerance.tck.config.fallback;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+
+/**
+ * Test class for fallback configuration
+ * @author carlosdlr
+ */
+public class ConfigPropertyGlobalVsMethodTest extends Arquillian {
+
+    @Inject
+    private BeanWithFallback bean;
+
+    @Deployment
+    public static WebArchive deployAnotherApp() {
+        JavaArchive testJar = ShrinkWrap
+            .create(JavaArchive.class, "ftConfigFallback.jar")
+            .addClasses(BeanWithFallback.class)
+            .addAsManifestResource(
+            new StringAsset(
+            "org.eclipse.microprofile.fault.tolerance.tck.config.fallback.BeanWithFallback/serviceA/Fallback/" +
+                "fallbackMethod=fallbackGlobalConfForServiceA"+
+                    "\norg.eclipse.microprofile.fault.tolerance.tck.config.fallback.BeanWithFallback/serviceA/Fallback/" +
+                "applyOn=RuntimeException.class"),
+                                   "microprofile-config.properties")
+            .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+            .as(JavaArchive.class);
+
+        return ShrinkWrap
+            .create(WebArchive.class, "ftConfigFallback.war")
+            .addAsLibrary(testJar);
+    }
+
+    /**
+     * Test that shows how the global(file configuration) has precedence over the annotation conf
+     */
+    @Test
+    void propertyGlobalConfAccessTest() {
+        try {
+            bean.serviceA();
+        }
+        catch (IllegalStateException e) {
+            e.printStackTrace();
+        }
+        Assert.assertEquals(bean.getCalledMethod(), "fallbackGlobalConfForServiceA");
```

Comment:
Better to use a return value, instead of modifying internal state? Also swallowing the exception is not nice :-)